.github/workflows/deploy.yml

```diff
@@ -85,7 +86,8 @@ jobs:
       - name: Create a release
         uses: softprops/action-gh-release@v1
         with:
-          name: ${{ steps.tagName.outputs.tag }}
+          name: v${{ steps.tagName.outputs.tag }}
+          body_path: CHANGELOG.txt
```

Comment:
I assume this automatically parses the Changelog for the version number and copies the release notes? That's neat.

Reply:
`CHANGELOG.txt` contains only the release notes, while the version is obtained using `cargo pkgid`